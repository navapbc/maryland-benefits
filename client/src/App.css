@import "~@cmsgov/design-system-core/dist/index.css";
@import "~@cmsgov/design-system-layout/dist/index.css";

.c-header__title {
  color: #fff;
  text-decoration: none;
}
.c-header__title:hover {
  color: #fff;
}

.c-body--wrapper {
  min-height: 300px;
}
