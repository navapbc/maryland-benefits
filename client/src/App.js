import React, { Component } from 'react';
import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
import Routes from './routes';

// CSS
import './App.css';

// Page Components
import Home from './components/Home';
import About from './components/About';
import Test from './components/Test';

class App extends Component {
  render() {
    return (
      <Router>
        <div className='app-component ds-base'>
          <header className='ds-base--inverse ds-u-padding--3 ds-u-display--flex ds-u-justify-content--between ds-u-align-items--center'>
            <h1 className='ds-h3 ds-u-margin-bottom--0'>
              <Link to='/' title='Home' className='c-header__title ds-u-text-decoration--none ds-u-color--base-inverse'>
                Maryland Benefits
              </Link>
            </h1>
          </header>

          <div className='ds-l-row ds-u-margin--0 c-body--wrapper'>
            <div>
              <Route
                exact
                path={Routes.home}
                component={Home}
              />
              <Route
                exact
                path={Routes.about}
                component={About}
              />
              <Route
                exact
                path={Routes.test}
                component={Test}
              />
            </div>
          </div>

          <footer className='ds-u-padding--3'>
            <div>
              <Link to='/' title='Home'>Home</Link>
              <Link to='/about' title='About' className='ds-u-padding--1'>About</Link>
            </div>

            <div className='ds-u-padding-y--3'>
              Brought to you by <a href='https://www.navapbc.com'>Nava PBC</a>
            </div>
          </footer>
        </div>
      </Router>
    );
  }
}

export default App;
