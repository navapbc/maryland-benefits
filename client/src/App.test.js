import React from 'react';
import ReactDOM from 'react-dom';
import { render } from './tests/TestHelper';

import App from './App';

describe('<App />', () => {
  it('renders', () => {
    const wrapper = render(<App/>);
    expect(wrapper).toMatchSnapshot();
  });
});
