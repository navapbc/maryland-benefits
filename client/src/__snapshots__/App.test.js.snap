// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <div
        className="app-component ds-base"
>
        <header
                className="ds-base--inverse ds-u-padding--3 ds-u-display--flex ds-u-justify-content--between ds-u-align-items--center"
        >
                <h1
                        className="ds-h3 ds-u-margin-bottom--0"
                >
                        <Link
                                className="c-header__title ds-u-text-decoration--none ds-u-color--base-inverse"
                                replace={false}
                                title="Home"
                                to="/"
                        >
                                Maryland Benefits
                        </Link>
                </h1>
        </header>
        <div
                className="ds-l-row ds-u-margin--0 c-body--wrapper"
        >
                <div>
                        <Route
                                component={[Function]}
                                exact={true}
                                path="/"
                        />
                        <Route
                                component={[Function]}
                                exact={true}
                                path="/about"
                        />
                        <Route
                                component={[Function]}
                                exact={true}
                                path="/test"
                        />
                </div>
        </div>
        <footer
                className="ds-u-padding--3"
        >
                <div>
                        <Link
                                replace={false}
                                title="Home"
                                to="/"
                        >
                                Home
                        </Link>
                        <Link
                                className="ds-u-padding--1"
                                replace={false}
                                title="About"
                                to="/about"
                        >
                                About
                        </Link>
                </div>
                <div
                        className="ds-u-padding-y--3"
                >
                        Brought to you by 
                        <a
                                href="https://www.navapbc.com"
                        >
                                Nava PBC
                        </a>
                </div>
        </footer>
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header
            className="ds-base--inverse ds-u-padding--3 ds-u-display--flex ds-u-justify-content--between ds-u-align-items--center"
>
            <h1
                        className="ds-h3 ds-u-margin-bottom--0"
            >
                        <Link
                                    className="c-header__title ds-u-text-decoration--none ds-u-color--base-inverse"
                                    replace={false}
                                    title="Home"
                                    to="/"
                        >
                                    Maryland Benefits
                        </Link>
            </h1>
</header>,
          <div
            className="ds-l-row ds-u-margin--0 c-body--wrapper"
>
            <div>
                        <Route
                                    component={[Function]}
                                    exact={true}
                                    path="/"
                        />
                        <Route
                                    component={[Function]}
                                    exact={true}
                                    path="/about"
                        />
                        <Route
                                    component={[Function]}
                                    exact={true}
                                    path="/test"
                        />
            </div>
</div>,
          <footer
            className="ds-u-padding--3"
>
            <div>
                        <Link
                                    replace={false}
                                    title="Home"
                                    to="/"
                        >
                                    Home
                        </Link>
                        <Link
                                    className="ds-u-padding--1"
                                    replace={false}
                                    title="About"
                                    to="/about"
                        >
                                    About
                        </Link>
            </div>
            <div
                        className="ds-u-padding-y--3"
            >
                        Brought to you by 
                        <a
                                    href="https://www.navapbc.com"
                        >
                                    Nava PBC
                        </a>
            </div>
</footer>,
        ],
        "className": "app-component ds-base",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1
              className="ds-h3 ds-u-margin-bottom--0"
>
              <Link
                            className="c-header__title ds-u-text-decoration--none ds-u-color--base-inverse"
                            replace={false}
                            title="Home"
                            to="/"
              >
                            Maryland Benefits
              </Link>
</h1>,
            "className": "ds-base--inverse ds-u-padding--3 ds-u-display--flex ds-u-justify-content--between ds-u-align-items--center",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="c-header__title ds-u-text-decoration--none ds-u-color--base-inverse"
                replace={false}
                title="Home"
                to="/"
>
                Maryland Benefits
</Link>,
              "className": "ds-h3 ds-u-margin-bottom--0",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Maryland Benefits",
                "className": "c-header__title ds-u-text-decoration--none ds-u-color--base-inverse",
                "replace": false,
                "title": "Home",
                "to": "/",
              },
              "ref": null,
              "rendered": "Maryland Benefits",
              "type": [Function],
            },
            "type": "h1",
          },
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div>
              <Route
                            component={[Function]}
                            exact={true}
                            path="/"
              />
              <Route
                            component={[Function]}
                            exact={true}
                            path="/about"
              />
              <Route
                            component={[Function]}
                            exact={true}
                            path="/test"
              />
</div>,
            "className": "ds-l-row ds-u-margin--0 c-body--wrapper",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <Route
                  component={[Function]}
                  exact={true}
                  path="/"
/>,
                <Route
                  component={[Function]}
                  exact={true}
                  path="/about"
/>,
                <Route
                  component={[Function]}
                  exact={true}
                  path="/test"
/>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "component": [Function],
                  "exact": true,
                  "path": "/",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "component": [Function],
                  "exact": true,
                  "path": "/about",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "component": [Function],
                  "exact": true,
                  "path": "/test",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                <Link
                                replace={false}
                                title="Home"
                                to="/"
                >
                                Home
                </Link>
                <Link
                                className="ds-u-padding--1"
                                replace={false}
                                title="About"
                                to="/about"
                >
                                About
                </Link>
</div>,
              <div
                className="ds-u-padding-y--3"
>
                Brought to you by 
                <a
                                href="https://www.navapbc.com"
                >
                                Nava PBC
                </a>
</div>,
            ],
            "className": "ds-u-padding--3",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Link
                    replace={false}
                    title="Home"
                    to="/"
>
                    Home
</Link>,
                  <Link
                    className="ds-u-padding--1"
                    replace={false}
                    title="About"
                    to="/about"
>
                    About
</Link>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Home",
                    "replace": false,
                    "title": "Home",
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": "Home",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "About",
                    "className": "ds-u-padding--1",
                    "replace": false,
                    "title": "About",
                    "to": "/about",
                  },
                  "ref": null,
                  "rendered": "About",
                  "type": [Function],
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Brought to you by ",
                  <a
                    href="https://www.navapbc.com"
>
                    Nava PBC
</a>,
                ],
                "className": "ds-u-padding-y--3",
              },
              "ref": null,
              "rendered": Array [
                "Brought to you by ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Nava PBC",
                    "href": "https://www.navapbc.com",
                  },
                  "ref": null,
                  "rendered": "Nava PBC",
                  "type": "a",
                },
              ],
              "type": "div",
            },
          ],
          "type": "footer",
        },
      ],
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <div
          className="app-component ds-base"
>
          <header
                    className="ds-base--inverse ds-u-padding--3 ds-u-display--flex ds-u-justify-content--between ds-u-align-items--center"
          >
                    <h1
                              className="ds-h3 ds-u-margin-bottom--0"
                    >
                              <Link
                                        className="c-header__title ds-u-text-decoration--none ds-u-color--base-inverse"
                                        replace={false}
                                        title="Home"
                                        to="/"
                              >
                                        Maryland Benefits
                              </Link>
                    </h1>
          </header>
          <div
                    className="ds-l-row ds-u-margin--0 c-body--wrapper"
          >
                    <div>
                              <Route
                                        component={[Function]}
                                        exact={true}
                                        path="/"
                              />
                              <Route
                                        component={[Function]}
                                        exact={true}
                                        path="/about"
                              />
                              <Route
                                        component={[Function]}
                                        exact={true}
                                        path="/test"
                              />
                    </div>
          </div>
          <footer
                    className="ds-u-padding--3"
          >
                    <div>
                              <Link
                                        replace={false}
                                        title="Home"
                                        to="/"
                              >
                                        Home
                              </Link>
                              <Link
                                        className="ds-u-padding--1"
                                        replace={false}
                                        title="About"
                                        to="/about"
                              >
                                        About
                              </Link>
                    </div>
                    <div
                              className="ds-u-padding-y--3"
                    >
                              Brought to you by 
                              <a
                                        href="https://www.navapbc.com"
                              >
                                        Nava PBC
                              </a>
                    </div>
          </footer>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <header
              className="ds-base--inverse ds-u-padding--3 ds-u-display--flex ds-u-justify-content--between ds-u-align-items--center"
>
              <h1
                            className="ds-h3 ds-u-margin-bottom--0"
              >
                            <Link
                                          className="c-header__title ds-u-text-decoration--none ds-u-color--base-inverse"
                                          replace={false}
                                          title="Home"
                                          to="/"
                            >
                                          Maryland Benefits
                            </Link>
              </h1>
</header>,
            <div
              className="ds-l-row ds-u-margin--0 c-body--wrapper"
>
              <div>
                            <Route
                                          component={[Function]}
                                          exact={true}
                                          path="/"
                            />
                            <Route
                                          component={[Function]}
                                          exact={true}
                                          path="/about"
                            />
                            <Route
                                          component={[Function]}
                                          exact={true}
                                          path="/test"
                            />
              </div>
</div>,
            <footer
              className="ds-u-padding--3"
>
              <div>
                            <Link
                                          replace={false}
                                          title="Home"
                                          to="/"
                            >
                                          Home
                            </Link>
                            <Link
                                          className="ds-u-padding--1"
                                          replace={false}
                                          title="About"
                                          to="/about"
                            >
                                          About
                            </Link>
              </div>
              <div
                            className="ds-u-padding-y--3"
              >
                            Brought to you by 
                            <a
                                          href="https://www.navapbc.com"
                            >
                                          Nava PBC
                            </a>
              </div>
</footer>,
          ],
          "className": "app-component ds-base",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h1
                className="ds-h3 ds-u-margin-bottom--0"
>
                <Link
                                className="c-header__title ds-u-text-decoration--none ds-u-color--base-inverse"
                                replace={false}
                                title="Home"
                                to="/"
                >
                                Maryland Benefits
                </Link>
</h1>,
              "className": "ds-base--inverse ds-u-padding--3 ds-u-display--flex ds-u-justify-content--between ds-u-align-items--center",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  className="c-header__title ds-u-text-decoration--none ds-u-color--base-inverse"
                  replace={false}
                  title="Home"
                  to="/"
>
                  Maryland Benefits
</Link>,
                "className": "ds-h3 ds-u-margin-bottom--0",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Maryland Benefits",
                  "className": "c-header__title ds-u-text-decoration--none ds-u-color--base-inverse",
                  "replace": false,
                  "title": "Home",
                  "to": "/",
                },
                "ref": null,
                "rendered": "Maryland Benefits",
                "type": [Function],
              },
              "type": "h1",
            },
            "type": "header",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div>
                <Route
                                component={[Function]}
                                exact={true}
                                path="/"
                />
                <Route
                                component={[Function]}
                                exact={true}
                                path="/about"
                />
                <Route
                                component={[Function]}
                                exact={true}
                                path="/test"
                />
</div>,
              "className": "ds-l-row ds-u-margin--0 c-body--wrapper",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Route
                    component={[Function]}
                    exact={true}
                    path="/"
/>,
                  <Route
                    component={[Function]}
                    exact={true}
                    path="/about"
/>,
                  <Route
                    component={[Function]}
                    exact={true}
                    path="/test"
/>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "component": [Function],
                    "exact": true,
                    "path": "/",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "component": [Function],
                    "exact": true,
                    "path": "/about",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "component": [Function],
                    "exact": true,
                    "path": "/test",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div>
                  <Link
                                    replace={false}
                                    title="Home"
                                    to="/"
                  >
                                    Home
                  </Link>
                  <Link
                                    className="ds-u-padding--1"
                                    replace={false}
                                    title="About"
                                    to="/about"
                  >
                                    About
                  </Link>
</div>,
                <div
                  className="ds-u-padding-y--3"
>
                  Brought to you by 
                  <a
                                    href="https://www.navapbc.com"
                  >
                                    Nava PBC
                  </a>
</div>,
              ],
              "className": "ds-u-padding--3",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <Link
                      replace={false}
                      title="Home"
                      to="/"
>
                      Home
</Link>,
                    <Link
                      className="ds-u-padding--1"
                      replace={false}
                      title="About"
                      to="/about"
>
                      About
</Link>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Home",
                      "replace": false,
                      "title": "Home",
                      "to": "/",
                    },
                    "ref": null,
                    "rendered": "Home",
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "About",
                      "className": "ds-u-padding--1",
                      "replace": false,
                      "title": "About",
                      "to": "/about",
                    },
                    "ref": null,
                    "rendered": "About",
                    "type": [Function],
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    "Brought to you by ",
                    <a
                      href="https://www.navapbc.com"
>
                      Nava PBC
</a>,
                  ],
                  "className": "ds-u-padding-y--3",
                },
                "ref": null,
                "rendered": Array [
                  "Brought to you by ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Nava PBC",
                      "href": "https://www.navapbc.com",
                    },
                    "ref": null,
                    "rendered": "Nava PBC",
                    "type": "a",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "footer",
          },
        ],
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
